README.md

```diff
@@ -160,11 +161,11 @@ The `scripts` directory contains scripts that can be used to test the lambda fun
 
 ### Local PSP beacon update
 
-First you need to start a local node and grab private keys of funded test accounts to add them to each account in the
-psp-beacon-local.json config file.
+First you need to start a local ethereum node and grab private keys of funded test accounts to add them to each account
+in the psp-beacon-local.json config file. Then you need to start the local 'Currency Converter API' server.
```

Comment:
The API must be started as well, right?

Reply:
You don't need it to run the script that sets everything up but you do need an API server to update beacons by running the handlers locally.